### PR TITLE
Improve metrics test

### PR DESCRIPTION
**Pull request by tmathern**
Fixes a failure on Windows machines running unit tests (travis may still fail).

### CODE REVIEW

File: test/metrics.test.js [hunk 1]
```diff
@@ -33,9 +33,7 @@ describe("metrics", () => {
     describe("openwhisk", () => {
 
         it("should return nodeVersion metric", () => {
-            assert.deepStrictEqual(Metrics.openwhisk(), {
-                nodeVersion: process.version.substr(1)
-            });
+            assert.strictEqual(Metrics.openwhisk().nodeVersion, process.version.substr(1));
```
Comment (alexkli) [inline test/metrics.test.js:36]: This change makes sense (more readable), but I wonder how this would fail on Windows?

Comment (tmathern) [reply test/metrics.test.js:36]: Looks like there is serialization going on, and the comparision fails because of the string quotes which are different...

Comment (alexkli) [reply test/metrics.test.js:36]: Hmm, this would be weird of `deepStringEqual` to do...

Comment (tmathern) [reply test/metrics.test.js:36]: It's the only difference I saw... But maybe I missed something else, like spaces, or endline characters.

**Pull request merged**